modules/sso-saml-provider/main.tf

```diff
@@ -0,0 +1,16 @@
+locals {
+  enabled = module.this.enabled
+
+  url    = try(module.store_read.map[format("%s/%s", var.ssm_path_prefix, "url")], "")
+  ca     = try(module.store_read.map[format("%s/%s", var.ssm_path_prefix, "ca")], "")
+  issuer = try(module.store_read.map[format("%s/%s", var.ssm_path_prefix, "issuer")], "")
+}
+
+module "store_read" {
```

Comment:
```suggestion
module "saml_store_read" {
```

Reply:
Naturally, other lines of code will change if you rename this, but I feel it could improve context when reading this code or searching for other code in a large directory of modules.